10.9-libcxx/stable/main/finkinfo/devel/logilab-common-py.info

```diff
@@ -1,9 +1,9 @@
 Info2: <<
 
 Package: logilab-common-py%type_pkg[python]
-Type: python (2.7 3.4 3.5 3.6)
+Type: python (2.7 3.4 3.5 3.6 3.7)
 Version: 0.63.2
-Revision: 1
+Revision: 2
 Source: http://pypi.python.org/packages/source/l/logilab-common/logilab-common-%v.tar.gz
 Source-MD5: 2bf4599ae1f2ccf4603ca02c5d7e798e
```

Comment:
logilab-common-py27-py35 has a bunch of syntax errors when building v1.7.0. Since the latest for both logilab-common and astroid-py have issues with older pythons, is it easier to include here only the build cleanups (and variant additions) then focus on the latest versions in a separate PR?